assets/js/components/user-input/UserInputQuestionWrapper.js

```diff
@@ -81,7 +84,7 @@ export default function UserInputQuestionWrapper( props ) {
 
 					{ isActive && (
 						<div className="googlesitekit-user-input__buttons">
-							{ back && (
+							{ ! single && back && (
```

Comment:
Let's use the same approach as we use for the next callback in the `UserInputQuestionnaire` component:

```javascript
	// Update the callbacks and labels for the questions if the user is editing a *single question*.
	let backCallback = back;
	let nextCallback = next;
	let nextLabel;

	if ( single === 'user-input' ) {
		// When the user is editing a single question in the user-input screen send them back to the preview when they click Update.
		nextCallback = goToPreview;
		nextLabel = __( 'Update', 'google-site-kit' );
		backCallback = undefined;
	} else if ( single === 'settings' ) {
		// When the user is editing a single question from the settings screen, submit changes and send them back to the settings pages when they click Submit.
		nextCallback = submitChanges;
		nextLabel = __( 'Submit', 'google-site-kit' );
		backCallback = undefined;
	}
```

In this case we can escape changing the wrapper component at all.